### ISSUE TITLE
Publish NPM 

### ISSUE THREAD

**Issue by thebergamo** [rafaeljesus/pagarme-node#2]
Hey man! Your package is pretty cool! Publish it on NPM :)

**Comment by rafaeljesus** [comment]
Hey @thebergamo thx a lot for contributing, I want to refactor to Promise based instead if callback before publish to NPM

**Comment by thebergamo** [comment]
Yeah! After finished implement in my product, I will help to refactoring that :)

**Comment by rafaeljesus** [comment]
@thebergamo what if we would use babel? tell me your thoughts
